assets/enhancements.css

```
@@ -141,15 +709,1868 @@ pre::-webkit-scrollbar-thumb:hover, .code-body::-webkit-scrollbar-thumb:hover{ba
+  scroll-margin: 90px;
+}
+.codecard.highlight-pulse,
+.codeblock.highlight-pulse,
+.viz.highlight-pulse,
+figure.highlight-pulse {
+  position: relative !important;
+  z-index: 10 !important;
+}
+@keyframes masterclassGlowPulse {
+  0% {
+    outline: 3px solid var(--accent) !important;
```

Stylelint (17.14.0)

[error] Disallowed !important (keyframe-declaration-no-important)

(keyframe-declaration-no-important)